backend/src/test/java/com/bakdata/conquery/integration/tests/deletion/ImportDeletionTest.java

```diff
@@ -177,29 +176,13 @@ public void execute(String name, TestConquery testConquery) throws Exception {
 			FileUtils.copyInputStreamToFile(In.resource(path.substring(0, path.lastIndexOf("/")) + "/" + "content2.2.csv")
 											  .asStream(), new File(conquery.getTmpDir(), import2Table.getCsv().getName()));
 
-			//create import descriptor
-			InputFile inputFile = InputFile.fromName(conquery.getConfig().getPreprocessor().getDirectories()[0], importId.getTag());
-			ImportDescriptor desc = new ImportDescriptor();
-			desc.setInputFile(inputFile);
-			desc.setName(import2Table.getName() + "_import");
-			desc.setTable(import2Table.getName());
-			Input input = new Input();
-			{
-				input.setPrimary(IntegrationUtils.copyOutput(0, import2Table.getPrimaryColumn()));
-				input.setSourceFile(new File(inputFile.getCsvDirectory(), import2Table.getCsv().getName()));
-				input.setOutput(new Output[import2Table.getColumns().length]);
-				for (int i = 0; i < import2Table.getColumns().length; i++) {
-					input.getOutput()[i] = IntegrationUtils.copyOutput(i + 1, import2Table.getColumns()[i]);
-				}
-			}
-			desc.setInputs(new Input[]{input});
-			Jackson.MAPPER.writeValue(inputFile.getDescriptionFile(), desc);
 
 			//preprocess
 			conquery.preprocessTmp();
 
 			//import preprocessedFiles
-			conquery.getDatasetsProcessor().addImport(conquery.getDataset(), inputFile.getPreprocessedFile());
+
+			conquery.getDatasetsProcessor().addImport(conquery.getDataset(), Preprocessor.getTaggedVersion(new File(conquery.getTmpDir(), import2Table.getCsv().getName().substring(0, import2Table.getCsv().getName().lastIndexOf('.')) + EXTENSION_PREPROCESSED), null));
```

Comment:
Gibt es auch einen Test in dem `tag != null` ist?

Reply:
Dafür müssten wir so tief in die Tests eingreifen für ein convenience feature

Reply:
aber ich sollte die methode selber unit testen